modules/ROOT/pages/edifact-receive-read-settings.adoc

```diff
@@ -19,62 +19,88 @@ It can take up to five minutes or so for your configuration changes to sync with
 [[ack-settings]]
 == Acknowledgment Settings
 
-To configure acknowledgements:
+Optionally configure acknowledgment settings for each Inbound EDIFACT message flow. To configure acknowledgement settings:
 
-. Enable the *Send CONTRL functional acknowledgement when messages fail to route to a message flow* setting.
-. Select an endpoint where the functional acknowledgement message will be sent to the trading partner.
+*Vijay, please make sure these descriptions are correct.*
 
-The *Send CONTRL functional acknowledgement when messages fail to route to a message flow* setting provides a functional acknowledgment (the receipt and syntactical acceptability of a message) with the exchange. Functional acknowledgment settings are configured for each message flow and apply only when there is a failure to route the message to one of the message flows.
+* Enable *Send CONTRL technical acknowledgement for each transmission from <partner-name>*.
++
+This acknowledgement indicates whether Partner Manager received and accepted the message.
++
```

Comment:
@dhirsch1  - For EDIFACT, there is just one CONTRL acknowledgement. the configuration "CONTRL technical acknowledgement" is being removed from the UI with this release, and we should remove these lines from the Doc.